### PR TITLE
RC3s for 21.08, 22.2.4 and 23.0.1

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ <skjnldsv@protonmail.com>

### CODE REVIEW

File: tests/integration/features/beta.feature [hunk 1]
```diff
@@ -545,17 +545,17 @@ Feature: Testing the update scenario of beta releases
     When The request is sent
     Then The response is non-empty
     And Update to version "23.0.1.1" is available
```
Comment (nickvergessen) [inline tests/integration/features/beta.feature:547]: version was not increased?

File: config/config.php [hunk 2]
```diff
@@ -539,103 +539,103 @@
...
+9psB9VX3BARdSiCT6qZbPmBZHpY9q+7YurYjJef0DBAmHWZ40RKRT0b44uEEytk7
+96rmSG0T48tzZaonZBXmgA==',
 			],
 		],
 
 		'20.0.14' => [
 			'100' => [
-				'latest' => '21.0.8 RC2',
+				'latest' => '21.0.8 RC3',
 				'internalVersion' => '21.0.8.1',
```
Comment (nickvergessen) [inline config/config.php:628]: same here, shouldn't the version be different?

Comment (skjnldsv) [reply config/config.php:628]: No, we didn't change in between.

Comment (skjnldsv) [reply config/config.php:628]: We blocked rc2, so I figured it was not need to

Comment (nickvergessen) [reply config/config.php:628]: That means people having 21.0.8 rc2 will not be offered the update to rc3
The numeric version must always increase

Comment (skjnldsv) [reply config/config.php:628]: We did not offered 21.0.8rc2

Comment (skjnldsv) [reply config/config.php:628]: https://github.com/nextcloud/updater_server/pull/687

Comment (nickvergessen) [reply config/config.php:628]: okay, right fine then.
Would just make it a fixed rule to be safe rather than sorry

**Review by nickvergessen — APPROVED** [review]

**Comment by nickvergessen** [comment]
This branch is out-of-date with the base branch
Merge the latest changes from master into this branch. 

😬

**Pull request merged**